ui/partials/perf/comparectrl.html

```diff
@@ -4,8 +4,9 @@
       Loading all results, please wait a minute...
       <img src="img/dancing_cat.gif" />
     </div>
-    <div id="error" ng-if="!dataLoading && errors.length" ng-repeat="error in errors">
-      <div>{{error}}</div>
+    <div id="error" ng-if="!dataLoading && errors.length">
+      <p ng-repeat="error in errors" class="bg-danger">{{error}}</p>
+      <a href="#/comparechooser?originalProject={{originalProject}}&originalRevision={{originalRevision}}&newProject={{newProject}}&newRevision={{newRevision}}">Modify choice</a>
```

Comment:
Ok cool, yeah, it's good to be consistent with the rest of Treeherder where we can. So I guess use `<span class="fa fa-warning"></span>` for the alert icon. Otherwise the rest of the advice still stands. :)